test/e2e/networkpolicy_test.go

```diff
@@ -344,12 +344,12 @@ func (data *TestData) setupDifferentNamedPorts(t *testing.T) (checkFn func(), cl
 // 2. The traffic initiated externally that access the Pod via NodePort service can be dropped (skipped if provider is kind).
 func testDefaultDenyIngressPolicy(t *testing.T, data *TestData) {
 	serverNode := workerNodeName(1)
-	serverNodeIP := workerNodeIP(1)
+	serverNodeIP := workerNodeIPv4(1)
```

Comment:
Can it still work with IPv6?

Reply:
I don't know. I modified this as I added a function called `workerNodeIPv6` to get Node IPv6.

Reply:
I believe it doesn't work. The test will also run with IPv6 only cluster. Previously the function returns IPv6 address in a IPv6 cluster, but you changed it to always return IPv4 address.
The only reason that it didn't fail CI was because the connection is expected to fail. And passing an empty server address also made the command fail.